--- /usr/src/mxe/src/boost.mk	2018-05-25 19:36:58.000000000 +0000
+++ boost.mk	2018-06-01 08:01:37.191623638 +0000
@@ -50,8 +50,6 @@
         cxxflags=$(if $(findstring posix,$(MXE_GCC_THREADS)),-std=gnu++11,-std=gnu++98) \
         --layout=tagged \
         --disable-icu \
-        --without-mpi \
-        --without-python \
         --prefix='$(PREFIX)/$(TARGET)' \
         --exec-prefix='$(PREFIX)/$(TARGET)/bin' \
         --libdir='$(PREFIX)/$(TARGET)/lib' \
